src/encoded/static/scss/encoded/modules/_matrix.scss

```diff
@@ -15,6 +18,390 @@ $row-data-header-width: 200px;
     background: linear-gradient(90deg, rgba(0, 0, 0, 0.1), rgba(0, 0, 0, 0.0));
 }
 
+.epigenome-body-map-container {
+    border: 2px solid $Homo-sapiens;
+    position: relative;
+    .body-list-top {
+        border-top: 0;
+    }
+    &.expanded {
+        max-width: 675px;
+        position: absolute;
+        z-index: 99999;
```

Comment:
Is there a reason why you are making the z-index to the max?

Reply:
This is the pop-up. It needs to be in front! I can make it less than the max though.